src/renderer/views/Channel/Channel.js

```diff
@@ -611,6 +655,12 @@ export default defineComponent({
           this.location = null
         }
 
+        const lowerTabs = channel.tabs.map(tab => tab.toLowerCase())
```

Comment:
Instead of relying on these localised strings, I would prefer it if you used YouTube.js' `has_{tabName}` getters, which check the URLs instead of the tab titles. I know it's more code but it's safer from YouTube unexpectedly changing things.
Feel free to store the results in variables so you don't have to call them again below.